omod/src/main/webapp/pages/results.gsp

```diff
@@ -156,7 +218,14 @@ def prettify = { "$it".replaceAll("null", "").replaceAll("\\s*,\\s", " ") }
         patients.push("${prettify(ui.encodeHtmlContent(it.name))}");
     </script>
     <tr>
-        <td style="display: none">rule_${it.rule.ruleId}</td>
+        <td class="hidden">rule_${it.rule.ruleId}</td>
+
+        <% if (it.patientProgram) { %>
+            <td>${it.patient_program_id}</td>
+        <% } else { %>
+            <td>${it.patient.patientIdentifier}</td>
+        <% } %>
+
         <td>${ui.encodeHtmlContent(it.rule.ruleName)}</td>
         <% if (it.patientProgram) { %>
```

Comment:
@shivtej1505 See comment above - the checks here should be inside a td since all they are doing is determining what content to display not whether to show the td